src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Log Exporter modified to no longer prefix scope-depth when exporting ILogger
+  scopes as attributes. Empty keys and {OriginalFormat} key will be ignored from
+  scopes.
```

Comment:
I'm not opposed to this change, but can you give a little more background on why we're making this change?

Reply:
Main motivation is the duplication/de-duplication cost, and undesired modification of user attributes. 
Explanation for each of the case:

1. With plain string, the key is null/empty, which is unallowed in OTel. But even if we were to use some magic key like "scope", it'll be duplicated, when user have nested scopes. The .NET official docs are being fixed already to *no longer* showing scopes with strings.

2. With {OriginalFormat} - this is an unintentional side effect in Ms.Ext.Logging. It makes sense for Log message, where there is a Format() function, which can be used to get the fully formatted string. There is nothing like that for Scopes. "{OriginalFormat}" key will be duplicated if user has nested scopes, and we do not want to pay for dedup efforts.

The current approach simply prefixed a numeric value (based on scope depth), to solve this problem. But overriding user provide attributes with something else is undesirable (and affects perf too). 

The following example can show why this is bad:

using (logger.BeginScope(new List<KeyValuePair<string, object>>
{
    new KeyValuePair<string, object>("userid", "cijo")
}))
using (logger.BeginScope(new List<KeyValuePair<string, object>>
{
    new KeyValuePair<string, object>("transactionid", "abcd")
}))
{
    logger.LogInformation("Hello from {food} {price}.", "artichoke", 3.99);
}

the above would have resulted in attributes ("[0]:userid", "cijo"), ("[1]:transactionid", "abcd") as per current code, instead of ("userid", "cijo").  ("transactionid", "abcd"), per this PR.